mmaction/core/evaluation/ava_evaluation/README.md

```diff
@@ -0,0 +1 @@
+The code under this folder is from the official [ActivityNet repo](https://github.com/activitynet/ActivityNet).
```

Comment:
Codes are not modified, right?

Reply:
Yes. However, some unused codes are removed to minimize the length of codes added.

Reply:
Okay, adding this note would be helpful for readers